Fabulous.XamarinForms/src/Fabulous.XamarinForms.Core/ViewConverters.fs

```diff
@@ -830,4 +830,19 @@ module Converters =
         | ValueNone, ValueNone -> ()
         | _, ValueSome currValue -> setUseSafeArea currValue
         | ValueSome _, ValueNone -> setUseSafeArea false
+
+    let updateEffects (prevValueOpt: string[] voption) (currValueOpt: string[] voption) (target: Xamarin.Forms.Element) =
```

Comment:
I think you can call `updateCollectionGeneric` directly in this function, instead of writing this pattern matching.
Currently this function doesn't clean up removed effects when the new value is not null.

Example:
`prev = View.Element(effects=["A"; "B"])`
`curr = View.Element(effects=["A"])`

Reply:
yes. forgot about it and currently trying to make it work with updatecollectiongeneric. 

Have a little problem on my side that "string is not compatible with Viewelement".

I thjink it is because my createfunction returns an Effect and not a ViewElement.Effect

Do you think we should just make a ViewElement.effect? 

Changed it to sth. like this:

```fsharp
    let updateEffects (prevValueOpt: string array voption) (currValueOpt: string array voption) (target: Xamarin.Forms.Element) =
        let createEffect effectName = Effect.Resolve effectName
        updateCollectionGeneric prevValueOpt currValueOpt target.Effects createEffect (fun _ _ _ -> ()) ViewHelpers.canReuseView updateChild
```

Reply:
>Do you think we should just make a ViewElement.effect?

Yes, seems like we should.
This will be necessary for @Happypig375's comment where effects can have parameters.

We can have a default `View.Effect(name=XXX)` that will represent an effect with no parameter.
Then a classic ViewElement creation will be used for effects with parameters.